l10n_br_stock_account/wizards/stock_return_picking.py

```diff
@@ -16,42 +16,41 @@ def _create_returns(self):
         """
         new_picking_id, pick_type_id = super()._create_returns()
 
-        if self.invoice_state == "2binvoiced":
+        picking_obj = self.env["stock.picking"]
+        picking = picking_obj.browse(new_picking_id)
 
-            picking_obj = self.env["stock.picking"]
-            move_obj = self.env["stock.move"]
+        origin_picking = self.env["stock.picking"].browse(self.env.context["active_id"])
+
+        if origin_picking.fiscal_operation_id:
 
-            origin_picking = self.env["stock.picking"].browse(
-                self.env.context["active_id"]
-            )
             refund_fiscal_operation = (
                 origin_picking.fiscal_operation_id.return_fiscal_operation_id
             )
 
             if not refund_fiscal_operation:
-                raise UserError(
-                    _(
-                        "This Fiscal Operation does not has Fiscal"
-                        " Operation for Returns!"
+                if self.invoice_state == "2binvoiced":
+                    raise UserError(
+                        _(
+                            "This Fiscal Operation does not has Fiscal"
```

Comment:
que tal corrigir o inglẽs com algo como: "This Fiscal Operation has no Fiscal Operation for Returns defined!"